test/test_processing_utils.py

```diff
@@ -13,41 +14,41 @@
 class ImagePreprocessing(unittest.TestCase):
     def test_decode_base64_to_image(self):
         output_image = gr.processing_utils.decode_base64_to_image(
-            gr.test_data.BASE64_IMAGE
+            media_data.BASE64_IMAGE
         )
         self.assertIsInstance(output_image, Image.Image)
 
     def test_encode_url_or_file_to_base64(self):
         output_base64 = gr.processing_utils.encode_url_or_file_to_base64(
             "test/test_data/test_image.png"
         )
-        self.assertEquals(output_base64, gr.test_data.BASE64_IMAGE)
+        self.assertEquals(output_base64, media_data.BASE64_IMAGE)
 
     def test_encode_file_to_base64(self):
         output_base64 = gr.processing_utils.encode_file_to_base64(
             "test/test_data/test_image.png"
         )
-        self.assertEquals(output_base64, gr.test_data.BASE64_IMAGE)
+        self.assertEquals(output_base64, media_data.BASE64_IMAGE)
 
     def test_encode_url_to_base64(self):
         output_base64 = gr.processing_utils.encode_url_to_base64(
             "https://raw.githubusercontent.com/gradio-app/gradio/master/test"
             "/test_data/test_image.png"
         )
-        self.assertEqual(output_base64, gr.test_data.BASE64_IMAGE)
+        self.assertEqual(output_base64, media_data.BASE64_IMAGE)
 
     # Commented out because this is throwing errors on Windows. Possibly due to different matplotlib behavior on Windows?
     # def test_encode_plot_to_base64(self):
     #     plt.plot([1, 2, 3, 4])
     #     output_base64 = gr.processing_utils.encode_plot_to_base64(plt)
-    #     self.assertEqual(output_base64, gr.test_data.BASE64_PLT_IMG)
+    #     self.assertEqual(output_base64, media_data.BASE64_PLT_IMG)
```

Comment:
@abidlabs I am commenting out this test while fixing the expected output value, it does not match.
```

E   AssertionError: 'data[107 chars]24zLjUuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/YYfK[23085 chars]QmCC' != 'data[107 chars]24zLjQuMywgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/MnkT[23085 chars]QmCC'
E   Diff is 46821 characters long. Set self.maxDiff to None to see it.
```